package.json

```diff
@@ -36,7 +36,7 @@
   ],
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "packages": "yarn workspaces foreach --from '{@unlock-protocol/types,@unlock-protocol/networks,@unlock-protocol/paywall,@unlock-protocol/unlock-js,@unlock-protocol/contracts}'",
+    "packages": "yarn workspaces foreach --from '{@unlock-protocol/types,@unlock-protocol/networks,@unlock-protocol/paywall,@unlock-protocol/unlock-js,@unlock-protocol/contracts,@unlock-protocol/ui}'",
```

Comment:
I wonder if there would be a way to use `@unlock-protocol/*`? Maybe @clemsos knows?

Reply:
Yeah, there is. 

But I think current way is fine. It allows us to keep packages that we don't use (for example, ui wasn't used until now) from building and causing unnecessary delay in getting things start for others.